### PR TITLE
update modules path in the reference file

### PR DESCRIPTION
**Pull request by kostasb**
modules reside under the modules.d directory
metricbeat.yml correctly sets path: ${path.config}/modules.d/*.yml
the reference file uses conf.d instead of modules.d

### CODE REVIEW

**Comment by ruflin** [comment]
Thanks for the fix. you will have to make the change here https://github.com/elastic/beats/blob/master/metricbeat/_meta/common.reference.yml#L17 and then run `make update` in the metricbeat directory.

**Comment by kostasb** [comment]
Let me try again under https://github.com/elastic/beats/pull/11516
